packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -75,6 +78,78 @@ export namespace TreeViewNode {
     }
 }
 
+export class ResolvableTreeViewNode implements TreeViewNode {
+    contextValue?: string | undefined;
```

Comment:
Minor: Remove `undefined` union type, for all properties below as well.
```suggestion
    contextValue?: string;
```